packages/mask/background/services/helper/popup-opener.ts

```diff
@@ -1,14 +1,14 @@
-import urlcat from 'urlcat'
-import { type DashboardRoutes, PopupRoutes, CrossIsolationMessages } from '@masknet/shared-base'
+import urlcat, { type ParamMap } from 'urlcat'
+import { type DashboardRoutes, PopupRoutes, MaskMessages, type PopupRoutesParamsMap } from '@masknet/shared-base'
 import { isLocked } from '../wallet/services/index.js'
 
 let currentPopupWindowId = 0
 
 async function openWindow(url: string): Promise<void> {
     const windows = await browser.windows.getAll()
-    const popup = windows.find((win) => win && win.type === 'popup' && win.id === currentPopupWindowId)
-    if (popup || currentPopupWindowId) {
-        await browser.windows.update(currentPopupWindowId, { focused: true })
+    const popup = windows.find((window) => window?.type === 'popup' && window.id === currentPopupWindowId)
+    if (popup) {
```

Comment:
Please don't remove the `currentPopupWindowId`, in some cases, `openWindow` will be called multiple times without throttling or debounce. Which will result in multiple windows being opened.